components/schemas.yaml

```diff
@@ -93,3 +93,100 @@ Method:
   # Option 2
   type: string
   default: 'TODO: override with own method name'
+
+# ================= Account Abstraction =================
+
+UserOperation:
+  type: object
+  properties:
+    sender:
+      type: string
+      description: The account making the operation
+      default: '0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045'
+      pattern: '^0[xX][0-9a-fA-F]+$'
+    nonce:
+      type: integer
+      description: Anti-replay parameter; also used as the salt for first-time account creation
+      default: 1
+    initCode:
+      type: string
+      description: The initCode of the account (needed if and only if the account is not yet on-chain and needs to be created)
+      default: '0x'
+    callData:
+      type: string
+      description: The data to pass to the sender during the main execution call
+      default: '0x'
```

Comment:
same feedback above, any chance we can get a real example?

Reply:
@elanh this is also a real example because if the transaction is not a contract interaction and someone is just transferring ETH using their smart contract wallet, this field will be `0x`. 

But can change it to the example of a contract interaction :)

Reply:
Changed to the example of a contract interaction `callData`